pizza-ingredients.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Ingredients</title>
+
+    <!-- import the webpage's stylesheet -->
+    <link rel="stylesheet" href="/ingredients.css" />
+
+    <!-- import the webpage's javascript file -->
+    <script src="/script.js" defer></script>
```

Comment:
Make sure your scripts are at the bottom of the page, just before the end `</body>` tag